.github/workflows/Dockerfile

```diff
@@ -49,7 +50,7 @@ RUN groupadd -g $GROUP_ID $GROUP && useradd -m -u $USER_ID -s /bin/sh -g $GROUP
 USER $USER:$GROUP
 
 RUN    pip3 install --user --upgrade pip  \
-    && ~/.local/bin/pip3 install --user lit
+    && ~/.local/bin/pip3 install --user lit pybind11
 
 ADD matching/pom.xml /home/user/.tmp-maven/
 RUN    cd /home/user/.tmp-maven \
```

Comment:
Is this working? It looks like compilation of `matching` is still downloading dependencies on the test action: https://github.com/runtimeverification/llvm-backend/actions/runs/8263962179/job/22606511215#step:5:4858

Reply:
Turns out this is expected behaviour; `dependency:go-offline` only caches build and test dependencies, not things like plugins.